packit/cli/dist_git_init.py

```diff
@@ -206,14 +205,21 @@ def init(
 
     Local generation for dist-git repo in current working directory:
 
+    \b
+
+        $ packit dist-git init
+
+    Local generation for dist-git repo in current working directory configuring git URL
+    for cloning of the upstream repository:
```

Comment:
I see that this is aligned with other descriptions but I found it difficult to understand. I would rewrite it, and the others, in a similar way as 
```suggestion
    Local packit configuration generation for a dist-git repo in the current working directory configuring git URL
    for cloning of the upstream repository:
```

Reply:
good idea, will do!